lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1256,6 +1256,7 @@ def deploy_vm(self):
                 if self.params['wait_for_ip_address']:
                     self.wait_for_vm_ip(vm)
 
+            self.change_detected = True
```

Comment:
I can add this line after **Clone** and **CreateVM_Task** but this two tasks are in the same function (deploy_vm) this is why I added this at the end of **deploy_vm**.
reconfigure_vm seems to already be ok.